main.tf

```diff
@@ -35,6 +35,7 @@ resource "aws_lambda_function" "lambda_asg" {
   handler          = "main.lambda_handler"
   runtime          = "python3.8"
   role             = aws_iam_role.lambda_role[0].arn
+  timeout          = 15
```

Comment:
@Davidoutz [minor] I would go even higher to stay on the safe side, smth like 30s doesn't hurt.

Reply:
Yes I was hesitating. I think it's the time it takes for boto to make to api call and get an answer from aws api. So it should not be that long. But it's more than the 3 default sec. I'll put 30 that's good. It failed probably because of that.